c2rust-analyze/src/known_fn.rs

```diff
@@ -143,7 +142,64 @@ impl Display for KnownFn {
     }
 }
 
-#[cfg(test)]
+impl KnownFn {
+    pub fn inputs_and_output(&self) -> impl Iterator<Item = &KnownFnTy> {
+        self.inputs.iter().chain([&self.output])
+    }
+
+    pub fn ptr_perms<'a, 'tcx>(
```

Comment:
can you add some documentation here explaining what's filtered and how the results are derived

Reply:
Added in e27102259801bd511c169829d6d6df03430f613b.